src/Sulu/Component/Content/Document/Subscriber/OrderSubscriber.php

```diff
@@ -64,19 +73,18 @@ public function handlePersist(PersistEvent $event)
 
         $propertyName = $this->encoder->systemName(self::FIELD);
 
-        if ($node->hasProperty($propertyName)) {
-            return;
-        }
-
         $parent = $node->getParent();
-        $nodeCount = count($parent->getNodes());
-        $order = ($nodeCount + 1) * 10;
+        $count = 0;
+        foreach ($parent->getNodes() as $childNode) {
```

Comment:
I guess it is always more efficient, so I would do the same for the `ReorderEvent`, especially because it is very confusing if the way to get the node changes all the time.